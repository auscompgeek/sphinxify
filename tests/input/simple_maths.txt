This is known to be broken. Output should include an escaped space before the role.

x<sub>1</sub>
x<sup>2</sup>
