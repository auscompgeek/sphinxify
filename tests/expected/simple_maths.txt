This is known to be broken. Output should include an escaped space before the role.

x:sub:`1`
x:sup:`2`
